WallPaperDev/WallPaperDev/Views/Welcome Views/WelcomeLabel.swift

```diff
@@ -9,6 +9,8 @@
 import UIKit
 
 class WelcomeLabel: UILabel {
+  private let fontSchema = DefaultFontSchema()
```

Comment:
Consider using `fontSchema = AppDependency.manager.currentTheme.fontSchema`

Reply:
Oh my bad, I didn't know that it was meant to be used like that 😅